src/util/deploy/get-deployment-by-url.ts

```diff
@@ -0,0 +1,14 @@
+import Client from '../client';
+import mapCertError from '../certs/map-cert-error';
+
+export default async function getDeploymentByUrl(client: Client, url: string) {
+  try {
+    return await client.fetch(url);
```

Comment:
This confused me at first, because it's not getting the deployment object but the actual content of the deployment, maybe we can rename this or just make this call without an extra function?

Reply:
@AndyBitz any recommendations for the naming - `getDeploymentContentsByUrl`?

Reply:
@mglagola yup, that sounds better

Reply:
@AndyBitz curl returns:

```json
{  
   "error":{  
      "message":"We could not create a certificate for external-test.xyz because HTTP pretest failed",
      "code":"configuration_error",
      "domain":"external-test.xyz",
      "external":false,
      "reason":"Invalid challenge response for external-test.xyz. Make sure the DNS records are correct.",
      "statusCode":449,
      "helpUrl":"https://err.sh/now-cli/cant-solve-challenge",
      "cns":[  
         "external-test.xyz"
      ]
   }
}
```